sitehost/server/datasource.go

```diff
@@ -0,0 +1,22 @@
+package server
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// DataSource returns a schema with the function to read Server resource.
+func DataSource() *schema.Resource {
+	recordSchema := serverDataSourceSchema()
+
+	return &schema.Resource{
+		ReadContext: readDataSource,
+		Schema:      recordSchema,
+	}
+}
+
+func readDataSource(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
```

Comment:
comment

should print `not implemented` so we know why it is empty

Reply:
```
func readDataSource(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
	fmt.Println("not implemented")
	return nil
}
``` like this?